app/Http/Requests/UserUpdateRequest.php

```diff
@@ -24,8 +26,23 @@ public function authorize()
     public function rules()
     {
         return [
-            'name' => 'required', 'string', 'max:255',
-            'email' => 'required', 'string', 'email', 'max:255', 'unique:users',
+            'name' => ['required', 'string', 'max:255'],
+            'email' => ['required', 'string', 'email', 'max:255'],
         ];
     }
+
+    public function getData() : array
+    {
+        $data = $this->only('name', 'email');
+
+        Validator::make($data, [
+            'email' => [
+                'required',
+                Rule::unique('users')->ignore($user->id),
+            ],
+        ]);
```

Comment:
Kindly remove. This should be on the email validation:
 'email' => ['required', 'string', 'email', 'max:255',  Rule::unique('users')->ignore($this->id)],